examples/_deployed/kitt/kitt.py

```diff
@@ -51,10 +75,58 @@ def on_chat_received(msg: rtc.ChatMessage):
 
         asyncio.create_task(_answer_from_text(msg.message))
 
+    async def respond_to_image(user_msg: str):
```

Comment:
would be good to have some comments in the example to walk users through the flow